.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
     "editor.formatOnSave": true,
-    "rust-client.channel": "nightly-2023-07-30",
-}
+    "rust-analyzer.server.extraEnv": {
+        "RUSTUP_TOOLCHAIN": "nightly-2023-07-30"
+    },
+    "rust-analyzer.check.allTargets": false,
+}
```

Comment:
?

Reply:
(why remove newline?)

Reply:
Was a case of `"files.insertFinalNewline": false` being the default in VSCode and `formatOnSave` just doing it's job. Should I add them back ? Additionally, I could add `"files.insertFinalNewline": true` to `settings.json` and override the user's default, to keep a rule.